install_config/oab_broker_configuration.adoc

```diff
@@ -553,6 +553,78 @@ is also required to configure the broker to use the Partner Registry URL:
     registry.connect.redhat.com
 ----
 
+[[oab-config-helm-chart-registry]]
+=== Helm Chart Registry
+
+Using the `helm` type allows you to consume Helm Charts from a Helm Chart Repository.
+
+[source,yaml]
+----
+registry:
+  - name: stable
+    type: helm
+    url: "https://kubernetes-charts.storage.googleapis.com"
+    runner: "docker.io/automationbroker/helm-runner:latest"
+    white_list:
+      - ".*"
+----
+
+*Note:* A heavy percentage of helm charts from the stable repository are not well suited for OpenShift.
+For more information see link:https://github.com/ansibleplaybookbundle/helm2bundle/wiki/Known-Issues[here].
+
+[[oab-config-api-v2-registry]]
+=== API V2 Docker Registry
+
+Using the `apiv2` type allows you to consume images from docker registries that implement the Docker Registry
+HTTP API V2 protocol can be configured with the apiv2 registry adapter.
+
+[source,yaml]
+----
+registry:
+  - name: <registry_name>
+    type: apiv2
+    url:  <registry_url>
+    user: <registry-user>
+    pass: <registry-password>
+    white_list:
+      - ".*-apb$"
+----
+
+If the registry requires authentication for pulling images, this can be achieved by running the following
+command on every node in your existing cluster:
+
+[source,yaml]
```

Comment:
This should not have source,yaml here since it's a terminal command. You can just remove this line.